### ISSUE TITLE
build.py:177: FutureWarning: The default value of regex will change from True to False in a future version.

### ISSUE THREAD

**Issue by mmp3** [micom-dev/micom#32]
Following your recommendation from #31 , I execute  `build_database` as follows:

```
# df has columns: 
#     id   kingdom   phylum  class  order  family   genus     species    file
> m = build_database( manifest = df , out_path = "/data/out" , threads = 8 )
```

which gives warning message (but still completes successfully):

```
/usr/local/lib/python3.8/dist-packages/micom/workflows/build.py:177: FutureWarning: The default value of regex will change from True to False in a future version.
  meta.index = meta[rank].str.replace("[^\\w\\_]", "_")
Running ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╺━━━━━━━━  79% 0:02:12
```

System information:

```
$ python3  -c "import micom; micom.show_versions()"

System Information
==================
OS                  Linux
OS-release 5.4.0-1030-aws
Python              3.8.5

Package Versions
================
cobra        0.21.0
jinja2       2.10.1
micom        0.22.5
pip          20.0.2
scikit-learn 0.24.1
scipy         1.6.1
setuptools   45.2.0
symengine     0.7.2
wheel        0.34.2
```

**Comment by cdiener** [comment]
Thanks! That's just a warning from pandas that some behavior will change in the future. It will not affect the model building at all. I'll fix those in one of the next releases though.

**Comment by cdiener** [comment]
Fixed in 0.22.7.

**Comment by nigiord** [comment]
Hello, apparently it's still there in 0.23.1 :

```
workflows/build.py:185: FutureWarning: The default value of regex will change from True to False in a future version.
  meta.index = meta[rank].str.replace("[^\\w\\_]", "_")
```
